v3/src/components/beta/unimplemented-menu-item.tsx

```diff
@@ -0,0 +1,14 @@
+import { MenuItem } from "@chakra-ui/react"
+import React from "react"
+
+interface IUnimplementedMenuItemProps {
+  label: string
+  testId?: string
+}
+export function UnimplementedMenuItem({ label, testId }: IUnimplementedMenuItemProps) {
+  return (
+    <MenuItem isDisabled data-testid={testId}>
+      {`🚧 ${label}`}
```

Comment:
```suggestion
      {`${label} 🚧`}
```
The convention elsewhere is to put the under construction symbol at the end of the label.